spec/numberformat.html

```diff
@@ -1595,14 +1719,20 @@ <h1>ToIntlMathematicalValue ( _value_ )</h1>
       </emu-alg>
     </emu-clause>
 
-    <emu-clause id="sec-getunsignedroundingmode" aoid="GetUnsignedRoundingMode">
-      <h1>GetUnsignedRoundingMode ( _roundingMode_, _isNegative_ )</h1>
-      <p>
-        The abstract operation GetUnsignedRoundingMode returns the rounding mode that should be applied to the absolute value of a number to produce the same result as if _roundingMode_, one of the String values in the *Identifier* column of <emu-xref href="#table-intl-rounding-modes"></emu-xref>, were applied to the signed value of the number (negative if the boolean _isNegative_ is *true*, or positive otherwise). The following steps are taken:
-      </p>
+    <emu-clause id="sec-getunsignedroundingmode" type="abstract operation">
+      <h1>
+        GetUnsignedRoundingMode (
+          _roundingMode_: a String,
+          _isNegative_: a Boolean,
+        ): a specification type from the Unsigned Rounding Mode column of <emu-xref href="#table-intl-unsigned-rounding-modes"></emu-xref>
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>It returns the rounding mode that should be applied to the absolute value of a number to produce the same result as if _roundingMode_, one of the String values in the Identifier column of <emu-xref href="#table-intl-rounding-modes"></emu-xref>, were applied to the signed value of the number (negative if the boolean _isNegative_ is *true*, or positive otherwise).</dd>
+      </dl>
       <emu-alg>
-        1. If _isNegative_ is *true*, return the specification type in the third column of <emu-xref href="#table-intl-unsigned-rounding-modes"></emu-xref> where the first column is _roundingMode_ and the second column is "negative".
-        1. Else, return the specification type in the third column of <emu-xref href="#table-intl-unsigned-rounding-modes"></emu-xref> where the first column is _roundingMode_ and the second column is "positive".
+        1. If _isNegative_ is *false*, let _sign_ be ~positive~; else let _sign_ be ~negative~.
```

Comment:
GetUnsignedRoundingMode is only invoked from FormatNumericToString AFAICT; let's just refactor the signature of the former from `_isNegative_: a Boolean` to e.g. `_sign_: ~negative~ or ~positive~`.
```suggestion
```
```patch
--- spec/numberformat.html
+++ spec/numberformat.html
@@ -759,10 +759,10 @@ FormatNumericToString (
         1. If _x_ is ~negative-zero~, then
-          1. Let _isNegative_ be *true*.
+          1. Let _sign_ be ~negative~.
           1. Set _x_ to 0.
         1. Else,
           1. Assert: _x_ is a mathematical value.
-          1. If _x_ &lt; 0, let _isNegative_ be *true*; else let _isNegative_ be *false*.
-          1. If _isNegative_ is *true*, then
+          1. If _x_ &lt; 0, let _sign_ be ~negative~; else let _sign_ be ~positive~.
+          1. If _sign_ is ~negative~, then
             1. Set _x_ to -_x_.
-        1. Let _unsignedRoundingMode_ be GetUnsignedRoundingMode(_intlObject_.[[RoundingMode]], _isNegative_).
+        1. Let _unsignedRoundingMode_ be GetUnsignedRoundingMode(_intlObject_.[[RoundingMode]], _sign_).
         1. If _intlObject_.[[RoundingType]] is ~significant-digits~, then
@@ -795,3 +795,3
           1. Set _string_ to the string-concatenation of _forwardZeros_ and _string_.
-        1. If _isNegative_ is *true*, then
+        1. If _sign_ is ~negative~, then
           1. If _x_ is 0, set _x_ to ~negative-zero~. Otherwise, set _x_ to -_x_.
```

Reply:
Fixed.